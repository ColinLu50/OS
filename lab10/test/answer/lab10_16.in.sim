56478
1645
863
833
857
830
