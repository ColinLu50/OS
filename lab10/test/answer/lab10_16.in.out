FCFS
26 -> 600 -> 39 -> 370 -> 735 -> 439 -> 57 -> 785 -> 488 -> 536 -> 530 -> 667 -> 831 -> 402 -> 571 -> 542 -> 501 -> 288 -> 837 -> 588 -> 478 -> 491 -> 90 -> 702 -> 471 -> 53 -> 189 -> 677 -> 795 -> 423 -> 97 -> 246 -> 466 -> 657 -> 282 -> 65 -> 821 -> 321 -> 744 -> 447 -> 397 -> 428 -> 79 -> 582 -> 86 -> 48 -> 459 -> 458 -> 572 -> 229 -> 84 -> 112 -> 47 -> 563 -> 289 -> 221 -> 49 -> 431 -> 219 -> 266 -> 761 -> 319 -> 168 -> 393 -> 507 -> 761 -> 677 -> 558 -> 763 -> 824 -> 26 -> 532 -> 612 -> 114 -> 95 -> 361 -> 625 -> 269 -> 742 -> 829 -> 581 -> 497 -> 746 -> 439 -> 244 -> 760 -> 650 -> 327 -> 282 -> 19 -> 122 -> 753 -> 94 -> 575 -> 256 -> 418 -> 158 -> 474 -> 755 -> 510 -> 224 -> 362 -> 481 -> 599 -> 3 -> 202 -> 722 -> 13 -> 646 -> 331 -> 594 -> 234 -> 492 -> 408 -> 668 -> 632 -> 37 -> 409 -> 638 -> 706 -> 359 -> 248 -> 154 -> 338 -> 148 -> 197 -> 596 -> 335 -> 403 -> 836 -> 439 -> 194 -> 521 -> 725 -> 31 -> 450 -> 256 -> 799 -> 820 -> 579 -> 677 -> 430 -> 51 -> 363 -> 652 -> 480 -> 646 -> 595 -> 803 -> 52 -> 746 -> 299 -> 344 -> 777 -> 244 -> 157 -> 293 -> 674 -> 368 -> 413 -> 265 -> 499 -> 13 -> 789 -> 261 -> 433 -> 48 -> 623 -> 515 -> 584 -> 72 -> 706 -> 624 -> 613 -> 537 -> 534 -> 504 -> 716 -> 516 -> 9 -> 346 -> 183 -> 602 -> 304 -> 821 -> 698 -> 601 -> 30 -> 742 -> 59 -> 67 -> 796 -> 204 -> 693 -> 520
56478
SSTF
26 -> 26 -> 30 -> 31 -> 37 -> 39 -> 47 -> 48 -> 48 -> 49 -> 51 -> 52 -> 53 -> 57 -> 59 -> 65 -> 67 -> 72 -> 79 -> 84 -> 86 -> 90 -> 94 -> 95 -> 97 -> 112 -> 114 -> 122 -> 148 -> 154 -> 157 -> 158 -> 168 -> 183 -> 189 -> 194 -> 197 -> 202 -> 204 -> 219 -> 221 -> 224 -> 229 -> 234 -> 244 -> 244 -> 246 -> 248 -> 256 -> 256 -> 261 -> 265 -> 266 -> 269 -> 282 -> 282 -> 288 -> 289 -> 293 -> 299 -> 304 -> 319 -> 321 -> 327 -> 331 -> 335 -> 338 -> 344 -> 346 -> 359 -> 361 -> 362 -> 363 -> 368 -> 370 -> 393 -> 397 -> 402 -> 403 -> 408 -> 409 -> 413 -> 418 -> 423 -> 428 -> 430 -> 431 -> 433 -> 439 -> 439 -> 439 -> 447 -> 450 -> 458 -> 459 -> 466 -> 471 -> 474 -> 478 -> 480 -> 481 -> 488 -> 491 -> 492 -> 497 -> 499 -> 501 -> 504 -> 507 -> 510 -> 515 -> 516 -> 520 -> 521 -> 530 -> 532 -> 534 -> 536 -> 537 -> 542 -> 558 -> 563 -> 571 -> 572 -> 575 -> 579 -> 581 -> 582 -> 584 -> 588 -> 594 -> 595 -> 596 -> 599 -> 600 -> 601 -> 602 -> 612 -> 613 -> 623 -> 624 -> 625 -> 632 -> 638 -> 646 -> 646 -> 650 -> 652 -> 657 -> 667 -> 668 -> 674 -> 677 -> 677 -> 677 -> 693 -> 698 -> 702 -> 706 -> 706 -> 716 -> 722 -> 725 -> 735 -> 742 -> 742 -> 744 -> 746 -> 746 -> 753 -> 755 -> 760 -> 761 -> 761 -> 763 -> 777 -> 785 -> 789 -> 795 -> 796 -> 799 -> 803 -> 820 -> 821 -> 821 -> 824 -> 829 -> 831 -> 836 -> 837 -> 19 -> 13 -> 13 -> 9 -> 3
1645
SCAN
26 -> 19 -> 13 -> 9 -> 3 -> 0 -> 30 -> 31 -> 37 -> 39 -> 47 -> 48 -> 49 -> 51 -> 52 -> 53 -> 57 -> 59 -> 65 -> 67 -> 72 -> 79 -> 84 -> 86 -> 90 -> 94 -> 95 -> 97 -> 112 -> 114 -> 122 -> 148 -> 154 -> 157 -> 158 -> 168 -> 183 -> 189 -> 194 -> 197 -> 202 -> 204 -> 219 -> 221 -> 224 -> 229 -> 234 -> 244 -> 246 -> 248 -> 256 -> 261 -> 265 -> 266 -> 269 -> 282 -> 288 -> 289 -> 293 -> 299 -> 304 -> 319 -> 321 -> 327 -> 331 -> 335 -> 338 -> 344 -> 346 -> 359 -> 361 -> 362 -> 363 -> 368 -> 370 -> 393 -> 397 -> 402 -> 403 -> 408 -> 409 -> 413 -> 418 -> 423 -> 428 -> 430 -> 431 -> 433 -> 439 -> 447 -> 450 -> 458 -> 459 -> 466 -> 471 -> 474 -> 478 -> 480 -> 481 -> 488 -> 491 -> 492 -> 497 -> 499 -> 501 -> 504 -> 507 -> 510 -> 515 -> 516 -> 520 -> 521 -> 530 -> 532 -> 534 -> 536 -> 537 -> 542 -> 558 -> 563 -> 571 -> 572 -> 575 -> 579 -> 581 -> 582 -> 584 -> 588 -> 594 -> 595 -> 596 -> 599 -> 600 -> 601 -> 602 -> 612 -> 613 -> 623 -> 624 -> 625 -> 632 -> 638 -> 646 -> 650 -> 652 -> 657 -> 667 -> 668 -> 674 -> 677 -> 693 -> 698 -> 702 -> 706 -> 716 -> 722 -> 725 -> 735 -> 742 -> 744 -> 746 -> 753 -> 755 -> 760 -> 761 -> 763 -> 777 -> 785 -> 789 -> 795 -> 796 -> 799 -> 803 -> 820 -> 821 -> 824 -> 829 -> 831 -> 836 -> 837
863
C-SCAN
26 -> 19 -> 13 -> 9 -> 3 -> 0 -> 837 -> 837 -> 836 -> 831 -> 829 -> 824 -> 821 -> 820 -> 803 -> 799 -> 796 -> 795 -> 789 -> 785 -> 777 -> 763 -> 761 -> 760 -> 755 -> 753 -> 746 -> 744 -> 742 -> 735 -> 725 -> 722 -> 716 -> 706 -> 702 -> 698 -> 693 -> 677 -> 674 -> 668 -> 667 -> 657 -> 652 -> 650 -> 646 -> 638 -> 632 -> 625 -> 624 -> 623 -> 613 -> 612 -> 602 -> 601 -> 600 -> 599 -> 596 -> 595 -> 594 -> 588 -> 584 -> 582 -> 581 -> 579 -> 575 -> 572 -> 571 -> 563 -> 558 -> 542 -> 537 -> 536 -> 534 -> 532 -> 530 -> 521 -> 520 -> 516 -> 515 -> 510 -> 507 -> 504 -> 501 -> 499 -> 497 -> 492 -> 491 -> 488 -> 481 -> 480 -> 478 -> 474 -> 471 -> 466 -> 459 -> 458 -> 450 -> 447 -> 439 -> 433 -> 431 -> 430 -> 428 -> 423 -> 418 -> 413 -> 409 -> 408 -> 403 -> 402 -> 397 -> 393 -> 370 -> 368 -> 363 -> 362 -> 361 -> 359 -> 346 -> 344 -> 338 -> 335 -> 331 -> 327 -> 321 -> 319 -> 304 -> 299 -> 293 -> 289 -> 288 -> 282 -> 269 -> 266 -> 265 -> 261 -> 256 -> 248 -> 246 -> 244 -> 234 -> 229 -> 224 -> 221 -> 219 -> 204 -> 202 -> 197 -> 194 -> 189 -> 183 -> 168 -> 158 -> 157 -> 154 -> 148 -> 122 -> 114 -> 112 -> 97 -> 95 -> 94 -> 90 -> 86 -> 84 -> 79 -> 72 -> 67 -> 65 -> 59 -> 57 -> 53 -> 52 -> 51 -> 49 -> 48 -> 47 -> 39 -> 37 -> 31 -> 30
833
LOOK
26 -> 19 -> 13 -> 9 -> 3 -> 30 -> 31 -> 37 -> 39 -> 47 -> 48 -> 49 -> 51 -> 52 -> 53 -> 57 -> 59 -> 65 -> 67 -> 72 -> 79 -> 84 -> 86 -> 90 -> 94 -> 95 -> 97 -> 112 -> 114 -> 122 -> 148 -> 154 -> 157 -> 158 -> 168 -> 183 -> 189 -> 194 -> 197 -> 202 -> 204 -> 219 -> 221 -> 224 -> 229 -> 234 -> 244 -> 246 -> 248 -> 256 -> 261 -> 265 -> 266 -> 269 -> 282 -> 288 -> 289 -> 293 -> 299 -> 304 -> 319 -> 321 -> 327 -> 331 -> 335 -> 338 -> 344 -> 346 -> 359 -> 361 -> 362 -> 363 -> 368 -> 370 -> 393 -> 397 -> 402 -> 403 -> 408 -> 409 -> 413 -> 418 -> 423 -> 428 -> 430 -> 431 -> 433 -> 439 -> 447 -> 450 -> 458 -> 459 -> 466 -> 471 -> 474 -> 478 -> 480 -> 481 -> 488 -> 491 -> 492 -> 497 -> 499 -> 501 -> 504 -> 507 -> 510 -> 515 -> 516 -> 520 -> 521 -> 530 -> 532 -> 534 -> 536 -> 537 -> 542 -> 558 -> 563 -> 571 -> 572 -> 575 -> 579 -> 581 -> 582 -> 584 -> 588 -> 594 -> 595 -> 596 -> 599 -> 600 -> 601 -> 602 -> 612 -> 613 -> 623 -> 624 -> 625 -> 632 -> 638 -> 646 -> 650 -> 652 -> 657 -> 667 -> 668 -> 674 -> 677 -> 693 -> 698 -> 702 -> 706 -> 716 -> 722 -> 725 -> 735 -> 742 -> 744 -> 746 -> 753 -> 755 -> 760 -> 761 -> 763 -> 777 -> 785 -> 789 -> 795 -> 796 -> 799 -> 803 -> 820 -> 821 -> 824 -> 829 -> 831 -> 836 -> 837
857
C-LOOK
26 -> 19 -> 13 -> 9 -> 3 -> 837 -> 836 -> 831 -> 829 -> 824 -> 821 -> 820 -> 803 -> 799 -> 796 -> 795 -> 789 -> 785 -> 777 -> 763 -> 761 -> 760 -> 755 -> 753 -> 746 -> 744 -> 742 -> 735 -> 725 -> 722 -> 716 -> 706 -> 702 -> 698 -> 693 -> 677 -> 674 -> 668 -> 667 -> 657 -> 652 -> 650 -> 646 -> 638 -> 632 -> 625 -> 624 -> 623 -> 613 -> 612 -> 602 -> 601 -> 600 -> 599 -> 596 -> 595 -> 594 -> 588 -> 584 -> 582 -> 581 -> 579 -> 575 -> 572 -> 571 -> 563 -> 558 -> 542 -> 537 -> 536 -> 534 -> 532 -> 530 -> 521 -> 520 -> 516 -> 515 -> 510 -> 507 -> 504 -> 501 -> 499 -> 497 -> 492 -> 491 -> 488 -> 481 -> 480 -> 478 -> 474 -> 471 -> 466 -> 459 -> 458 -> 450 -> 447 -> 439 -> 433 -> 431 -> 430 -> 428 -> 423 -> 418 -> 413 -> 409 -> 408 -> 403 -> 402 -> 397 -> 393 -> 370 -> 368 -> 363 -> 362 -> 361 -> 359 -> 346 -> 344 -> 338 -> 335 -> 331 -> 327 -> 321 -> 319 -> 304 -> 299 -> 293 -> 289 -> 288 -> 282 -> 269 -> 266 -> 265 -> 261 -> 256 -> 248 -> 246 -> 244 -> 234 -> 229 -> 224 -> 221 -> 219 -> 204 -> 202 -> 197 -> 194 -> 189 -> 183 -> 168 -> 158 -> 157 -> 154 -> 148 -> 122 -> 114 -> 112 -> 97 -> 95 -> 94 -> 90 -> 86 -> 84 -> 79 -> 72 -> 67 -> 65 -> 59 -> 57 -> 53 -> 52 -> 51 -> 49 -> 48 -> 47 -> 39 -> 37 -> 31 -> 30
830
